Lombiq.BaseTheme.Samples/Lombiq.BaseTheme.Samples.csproj

```diff
@@ -0,0 +1,28 @@
+<Project Sdk="Microsoft.NET.Sdk.Razor">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <AddRazorSupportForMvc>true</AddRazorSupportForMvc>
+    <DefaultItemExcludes>$(DefaultItemExcludes);.git*;node_modules\**;Tests\**</DefaultItemExcludes>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
```

Comment:
Are we sure about this?

Reply:
Sure, why not?

Reply:
After all, science is not about why, but why not.

I mean, this is something I'd govern at the solution-level, and with .NET Analyzers, after discussing it with the team.

Reply:
I don't think solution-level is practical becuase we don't want to retrofit everything in one go. MS made it a solution property this so teams with existing code base can introduce it progressively.

Reply:
OK, let's leave it, and follow up under https://github.com/Lombiq/.NET-Analyzers/issues/48.